packages/db/test/unit/controller/level.test.ts

```diff
@@ -113,6 +113,8 @@ describe("LevelDB controller", () => {
     expect(result.length).to.be.equal(2);
   });
 
+  // TODO Fails on any other platform part from Linux
+  // Issue here https://github.com/ChainSafe/lodestar/issues/4525
```

Comment:
Then add:
```ts
if (!isLinux) this.skip()
```

Reply:
I think a better approach is to fix the underlying issue. Instead of having platform dependent code that only asserts test behaviour for a particular platform.